### ISSUE TITLE
[sql] How to specify the directory when load a sqlite file.

### ISSUE THREAD

**Issue by chumomo** [tauri-apps/plugins-workspace#275]
Default is to save/load in roaming folder, how to change the base folder?
Thanks!

**Comment by FabianLars** [comment]
This is not yet possible. I did start working on it here https://github.com/tauri-apps/plugins-workspace/tree/sql/absolute-paths but had to focus on other things. With this change you'll be able to specify absolute paths and specify a different base directory (https://tauri.app/v1/api/js/fs/#basedirectory)

**Comment by yipjunkai** [comment]
Is there any update on this enchancement?

**Comment by FabianLars** [comment]
no

**Comment by alinnert** [comment]
Is this actually still planned? The link provided above doesn't work anymore. I'm wondering if I can start using the sql plugin now with paths relative to `...BaseDirectory::App` and migrate to custom paths later as soon as they get available.

Otherwise I'd need to look for an alternative approach. (What I need is basically some sort of file format that acts as a "document" the user can open with the app I'm developing. And SQLite would be great for that.)

**Comment by FabianLars** [comment]
Yes, still planned. Unforeseen life stuff happened so I never got to it but I hope I can spend some time on the plugin soon.

**Comment by alinnert** [comment]
Alright, no problem. Life comes first!

**Comment by fokx** [comment]
You can change the starting location by clone [plugins-workspace](https://github.com/tauri-apps/plugins-workspace/) to local, and change the following line
https://github.com/tauri-apps/plugins-workspace/blob/65a77b73d5e81b71faf130242f5b7c7230556ba4/plugins/sql/src/wrapper.rs#L120
to, e.g.,   ` .document_dir()`

then configure Cargo.toml and package.json to use it:
tauri-plugin-sql = { path = "../../plugins-workspace/plugins/sql", features = ["sqlite"] }
    "plugin-sql": "link:../plugins-workspace/plugins/sql",

**Comment by han4wluc** [comment]
Hi @FabianLars,

I really need the ability to use SQLite databases in paths outside of `tauri::api::path::BaseDirectory::AppConfig`.

## Current Behavior
The library expects the value after `:` to be a relative path:
- `sqlite:mydb.db` → `${APP_DIR}/mydb.db`

## Proposal
Enable passing absolute paths after `sqlite:`

### Examples:
- `sqlite:/home/user/mydb.db` → `/home/user/mydb.db` (Linux/macOS)
- `sqlite:C:\Users\data\mydb.db` → `C:\Users\data\mydb.db` (Windows)
- `sqlite:mydb.db` → `${APP_DIR}/mydb.db` (unchanged - relative path)

## Implementation Approach
Check if the path after `sqlite:` is absolute using `std::path::Path::is_absolute()`:
- If absolute, use it directly
- If relative, maintain the existing `path_mapper` behavior

The changes would be around here:
https://github.com/tauri-apps/plugins-workspace/blob/5e5bc30ad0672a35de97eeead80cc7a7bbf9cf2b/plugins/sql/src/wrapper.rs#L79

Is this what you had in mind? Or let me know your preferred approach, and I'd be happy to work on it.

Thanks!
